### ISSUE TITLE
invalid INS incorrect test

### ISSUE THREAD

**Issue by antonio-fr** [fido-alliance/conformance-tools-issues#544]
## What protocol and version of the protocol are you testing?

U2F 1.1

## What is your implementation class?

U2F token

## What is the version of the tool are you using?

fido-conformance-tools-electron-win32-x64 1.3.2

## What is the OS and the version are you running?
Windows 10

## Issue description

"NFC" mode with a CCID contact USB reader :

NFC-1 Test U2F NFC support

F-1 Send Register command with invalid INS, wrapped in Short APDU, and check that authenticator returns APDU error SW_INS_NOT_SUPPORTED(0x6D00) ‣
```
AssertionError: Expected to get SW_INS_NOT_SUPPORTED(0x6D00). Got SW_CLA_NOT_SUPPORTED(28160): expected 27904 to equal 28160
    at window.navigator.fido.fido2.nfc.sendNFCAPDUBuffers.catch (eval at compileCode (js/sandbox.js:25:26), <anonymous>:2270:24)
    at <anonymous>
```

F-2 Send Register command with invalid INS, wrapped in Extended APDU, and check that authenticator returns APDU error SW_INS_NOT_SUPPORTED(0x6D00) ‣
```
AssertionError: Expected to get SW_INS_NOT_SUPPORTED(0x6D00). Got SW_CLA_NOT_SUPPORTED(28160): expected 27904 to equal 28160
    at window.navigator.fido.fido2.nfc.sendNFCAPDUBuffers.catch (eval at compileCode (js/sandbox.js:25:26), <anonymous>:2282:24)
    at <anonymous>
    at <anonymous>
```

The conformance tool seems to send a bad CLA (different from 0x00), and the device returns 0x6E00 instead of 0x6D00. So the test are not passed because the command sent is not good for this test.

**Comment by herrjemand** [comment]
@antonio-fr Try latest build

**Comment by antonio-fr** [comment]
Still present in the latest build 1.3.4. I thought it could have been fix thanks to #536 , but it was probably fixed only in the HID part.

NFC communication channel with aCCID USB reader.

Here's the output with a very strict implementation, responding SW_CLA_NOT_SUPPORTED 0x6E00 when the CLA byte is different from 0.

**NFC-1 Test U2F NFC support**

F-1 Send Register command with invalid INS, wrapped in Short APDU, and check that authenticator returns APDU error SW_INS_NOT_SUPPORTED(0x6D00) ‣
```
AssertionError: Expected to get SW_INS_NOT_SUPPORTED(0x6D00). Got SW_CLA_NOT_SUPPORTED(28160): expected 27904 to equal 28160
    at window.navigator.fido.fido2.nfc.sendNFCAPDUBuffers.catch (eval at compileCode (js/sandbox.js:25:26), <anonymous>:2198:24)
    at <anonymous>
```
F-2 Send Register command with invalid INS, wrapped in Extended APDU, and check that authenticator returns APDU error SW_INS_NOT_SUPPORTED(0x6D00) ‣
```
AssertionError: Expected to get SW_INS_NOT_SUPPORTED(0x6D00). Got SW_CLA_NOT_SUPPORTED(28160): expected 27904 to equal 28160
    at window.navigator.fido.fido2.nfc.sendNFCAPDUBuffers.catch (eval at compileCode (js/sandbox.js:25:26), <anonymous>:2210:24)
    at <anonymous>
```


The logs (sending part) :
    
```
Test started: F-1
Send Register command with invalid INS, wrapped in Short APDU, and check that authenticator returns APDU error SW_INS_NOT_SUPPORTED(0x6D00)
transports\nfcdep.js:102 Selected protocol "2" for reader "ACS ACR122 0"
transports\nfcdep.js:218 NFC DATA SENT: 00a4040008a0000006472f000100
transports\nfcdep.js:223 NFC DATA RECEIVED: 5532465f56329000
transports\nfcdep.js:191 RECEIVED  SW_NO_ERROR
transports\nfcdep.js:218 NFC DATA SENT: 80750000407136c5d2c58718864ad7d360dbb2b82fc672ea78a3914c1bedf0627aa160e90110fd4a3b100310103c6b19b3c263f3beb02e7db7aec80015ffdd76d4e25f60ed00
...

controller.js:331 Test started: F-2
Send Register command with invalid INS, wrapped in Extended APDU, and check that authenticator returns APDU error SW_INS_NOT_SUPPORTED(0x6D00)
transports\nfcdep.js:102 Selected protocol "2" for reader "ACS ACR122 0"
transports\nfcdep.js:218 NFC DATA SENT: 00a4040008a0000006472f000100
transports\nfcdep.js:223 NFC DATA RECEIVED: 5532465f56329000
transports\nfcdep.js:191 RECEIVED  SW_NO_ERROR
transports\nfcdep.js:218 NFC DATA SENT: 80750000000040a8f97376a678f82d628e87ea48f21e0fe08f7798db5ef386d900361b04e96ef293f63519647ff0eca38bb78800ada2ec4b950cc3f8583af9d282acdbdc6892ec0000
...

```
Some famous U2F tokens are responding with 0x6d00 SW_INS_NOT_SUPPORTED for these commands, but strictly they should respond 0x6e00 SW_CLA_NOT_SUPPORTED because the given CLA is different from 0x00. We could argue that this CLA is supported by the token, as some latitude can be allowed in filtering the MSB bit of CLA. In our implementation, we choose to throw SW_CLA_NOT_SUPPORTED whenever the CLA is NOT 0x00.

In the logs, we can clearly see the sent CLA is 0x80, different from the expecting 0x00. The U2F standard states that "CLA : The host shall set this byte to zero". So anyway, the testing tool shall send 0x00.
